modules/common/src/main/java/com/sygic/maps/module/common/MapFragmentWrapper.kt

```diff
@@ -80,6 +78,10 @@ abstract class MapFragmentWrapper<T: ThemeSupportedViewModel> : MapFragment(), S
     @Inject
     internal lateinit var mapInteractionManager: MapInteractionManager
     @Inject
+    internal lateinit var mapDataModel: ExtendedMapDataModel
```

Comment:
nemůžeš je injectovat daggerem, protože pak ti přestanou fungovat metody jako add/remove marker... Nic ti přeci nebrání je zavolat hned po new BrowseMapFragment() a v tu chvilku je dagger ještě mimo hru...

Reply:
Nie, funguje to bez problemov vdaka Fragment scope

Reply:
takže když udělám:
```
val fragment = BrowseMapFragment()
fragment.addMapMarker(MapMarker(48.264514, 17.248591))
```
tak to necrashne? :-D kdo v té době ten slavný dagger s inicializuje? :-D

Reply:
lenze nemoze ich ani vytvorit fragment, lebo potom budu s kazdou rotaciou rovnake